content/en/blog/OWNERS

```diff
@@ -1,11 +1,17 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
-# Owned by Kubernetes Blog reviewers.
+# Owned by Kubernetes Blog reviewers
+
 options:
   no_parent_owners: false
```

Comment:
no_parent_owners is false by default, so this isn't really needed :)

Reply:
@nikhita :wave: There’s an open discussion about whether to reenable it in the next couple months. For now, it’s handy to have around—and we can always remove it in a separate PR.